frontend/src/components/SystemSubmitDrawer/ClientCodeDisplay.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { CopyBlock, dracula } from "react-code-blocks";
+
+export type CodeGenFields = {
+  task: string;
+  system_names: string[];
+  system_output_file_type: string;
+  dataset: string;
+  split: string;
+  source_language: string;
+};
+
+function getPythonClientCode(fields: CodeGenFields) {
+  return `
+    import os
+    import explainaboard_client
```

Comment:
Maybe add a simple docstring, for example:

# pip install explainaboard_client